### ISSUE TITLE
[QST] All_Simple_Path Cugraph

### ISSUE THREAD

**Issue by Algochain** [rapidsai/cugraph#2392]
Hello there ! I've seen in another issue that the NetworkX function all_simple_path was adapted for Cugraph in version 21.12
(Related issue : https://github.com/rapidsai/cugraph/issues/1854)

I've just installed RAPIDs and Cugraph with Docker. I see that my cugraph package is on version 21.12.0+0.g3a43e9d0.dirty.

However, i get this error when running my Python script : 
`AttributeError: module 'cugraph' has no attribute 'all_simple_paths'`

Here is the code I use : 

```
import cugraph as cnx
import networkx as nx
Graph = nx.MultiDiGraph()
# I add some edges here
cnx.all_simple_paths(Graph, source=source, target=target, cutoff=5)
```

**Comment by BradReesWork** [comment]
@Algochain we were hoping to have finished all_simple_paths but that effort has slipped.  There are a number of memory issues as the graph size increases that makes all paths inefficient.  If you graph is relatively small then there is a slight workaround using data replication.   See:  https://github.com/rapidsai/cugraph/blob/branch-22.08/notebooks/applications/CostMatrix.ipynb

We do get a lot of request for all_simple_path (ASSP) and it is still being worked, just not ready yet

**Comment by BradReesWork** [comment]
@Algochain for a million edges, it would need several passes.  The work-around works by creating multiple copies of the graph and then running a a SSSP on each graph from a different starting point.  To accomplish that we create a ghost, or dummy, node that links all starting points together.  Since our SSSP algorithm operates in parallel, starting with a ghost node allows all the second hops (true starting points) to run in parallel.  So doing X starting source nodes runs as fast as a single starting node.

**Comment by BradReesWork** [comment]
now it is not an all paths per source but a SSSP from every source
